plugins/diagnose.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: diagnose
```

Comment:
This `diagnose` name is sadly too generic.
It violates our "first comers advantage" rule in the Naming Guide.

As you might already see, we already have plenty of plugins in this repo that provide some form of "diagnosing" capabilities or another.

The exception to this has been the `debug` plugin which has now graduated into kubectl core. Other than that, we tend to reject names that put dibs on an entire class of verbs or resources. (e.g. `kubectl statefulset`, `kubectl service`, `kubectl inspect`...)

Reply:
@ahmetb 
Thats totally fair.

Would something like `kubectl poddiagnose` or `kubectl pod-diagnose` better fit? I can just rename it in the manifest to copy over as one of those

Reply:
Ideally we like dash between these words but there's precedent for both.
Pick and choose what’s better for your users, our Naming Guide recommends dashes in between.
I'll let @corneliusweig also chime in the proposed names.

Reply:
Yeah, `diagnose` is a bit too generic. `pod-diagnose` is certainly an option, but diagnose sounds more interactive to me. What about `sick-pods`? To me that sounds more like it would just find list unhealthy pods (which your plugin does, AFAIU).

Reply:
I like `sick-pods`! That fits the theme and does make a lot of sense. I'll change it to that 👍